--- deng-1.9.0-beta6.9/doomsday/plugins/jdoom/src/m_cheat.c~	2010-04-07 22:52:50.000000000 +0200
+++ deng-1.9.0-beta6.9/doomsday/plugins/jdoom/src/m_cheat.c	2013-11-07 14:11:30.116079643 +0100
@@ -489,7 +489,7 @@
     P_SetMessage(plr, textBuffer, false);
 
     // Also print some information to the console.
-    Con_Message(textBuffer);
+    Con_Message("%s", textBuffer);
     sub = plr->plr->mo->subsector;
     Con_Message("\nSubsector %i:\n", P_ToIndex(sub));
     Con_Message("  FloorZ:%g Material:%s\n",
--- deng-1.9.0-beta6.9/doomsday/plugins/common/src/p_xgline.c~	2010-04-07 22:52:50.000000000 +0200
+++ deng-1.9.0-beta6.9/doomsday/plugins/common/src/p_xgline.c	2013-11-07 14:12:20.268270732 +0100
@@ -424,7 +424,7 @@
     va_start(args, format);
     dd_vsnprintf(buffer, sizeof(buffer), format, args);
     strcat(buffer, "\n");
-    Con_Message(buffer);
+    Con_Message("%s", buffer);
     va_end(args);
 }
 
--- deng-1.9.0-beta6.9/doomsday/plugins/jheretic/src/m_cheat.c~	2010-04-07 22:52:50.000000000 +0200
+++ deng-1.9.0-beta6.9/doomsday/plugins/jheretic/src/m_cheat.c	2013-11-07 14:17:09.758211578 +0100
@@ -382,7 +382,7 @@
     P_SetMessage(plr, textBuffer, false);
 
     // Also print some information to the console.
-    Con_Message(textBuffer);
+    Con_Message("%s", textBuffer);
     sub = plr->plr->mo->subsector;
     Con_Message("\nSubsector %i:\n", P_ToIndex(sub));
     Con_Message("  FloorZ:%g Material:%s\n",
--- deng-1.9.0-beta6.9/doomsday/plugins/jhexen/src/m_cheat.c~	2010-04-07 22:52:50.000000000 +0200
+++ deng-1.9.0-beta6.9/doomsday/plugins/jhexen/src/m_cheat.c	2013-11-07 14:23:54.638714186 +0100
@@ -454,7 +454,7 @@
     P_SetMessage(plr, textBuffer, false);
 
     // Also print some information to the console.
-    Con_Message(textBuffer);
+    Con_Message("%s", textBuffer);
     sub = plr->plr->mo->subsector;
     Con_Message("\nSubsector %i:\n", P_ToIndex(sub));
     Con_Message("  FloorZ:%g Material:%s\n",
