--- deng-1.8.3.orig/configure.ac	2004-10-19 16:10:54.000000000 +0200
+++ deng-1.8.3/configure.ac	2004-10-30 01:00:38.248519448 +0200
@@ -57,7 +57,7 @@
 dnl	])
 
 dnl Check that the required libraries are available.
-AC_CHECK_HEADER(curses.h,
+AC_CHECK_HEADER(ncurses/curses.h,
 	[USELESS_VAR=1],
 	AC_MSG_ERROR([*** curses not found!]))
 
--- deng-1.8.3.orig/Src/Makefile.am	2004-10-19 15:50:52.000000000 +0200
+++ deng-1.8.3/Src/Makefile.am	2004-10-30 01:15:19.003624320 +0200
@@ -2,7 +2,7 @@
 
 bin_PROGRAMS = doomsday
 
-INCLUDES = -I$(top_srcdir)/Include
+INCLUDES = -I$(top_srcdir)/Include -I/usr/include/ncurses
 DEFS = -DUNIX -D@FIXED_IMPL@ -D__DOOMSDAY__ -DNORANGECHECKING	   \
 -DDENG_LIBRARY_DIR=\"$(libdir)\" -DDENG_BASE_DIR=\"$(pkgdatadir)\"
 doomsday_CFLAGS = @SDL_CFLAGS@ $(LTDLINCL)
